template/create_eu_codes.R

```diff
@@ -0,0 +1,11 @@
+# Get EU + UK country names + codes
+
+eu28 <- countrycode::codelist
+
+eu28 <- eu28[!is.na(eu28$eu28),]
```

Comment:
The plan is to have at least EU + EFTA + UK - see here for getting a list of iso3c codes for the relevant countries
https://github.com/epiforecasts/covid19-forecast-hub-europe/blob/35f5fe94d137a9848ed7a53880ab7ba8e002e9f8/code/auto_download/ecdc_download.r#L12